HOWTOSTART.md

```diff
@@ -0,0 +1,334 @@
+# Setting up Development Environment for Origam
+The following is a guide to help you set up Origam development environment. You will need it if you wish to contribute to Origam backend or frontend. It will be quite helpful if you are going to develop your own plugin for Origam too.
```

Comment:
I do not know how exactly the scope of this document is constrained - I just miss somewhere mentioned that you need specific versions of Origam (both the Server and mainly the Architect) for specific projects in case you want to work with something different from Demo. But opening another project is not mentioned as well so I think it is rather subject for different document.